libnmstate/nm/bond.py

```diff
@@ -45,9 +45,10 @@ def get_bond_info(nm_device):
 
 
 def get_options(nm_device):
-    con = connection.get_device_connection(nm_device)
-    if con:
-        bond_settings = con.get_setting_bond()
+    con = connection.ConnectionProfile()
+    con.import_by_device(nm_device)
+    if con.profile:
+        bond_settings = con.profile.get_setting_bond()
```

Comment:
In the future, the ConnectionProfile object should have a property with a BondSetting object that containsthe bond_settings and that should be used here.

Reply:
Yes, it should have a list of Setting objects, with concrete instances (like BondSetting).